.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Pytest with Flake8
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Recommend changing this to a matrix with Mac and Linux

Reply:
Running it now, but I recommend dropping the mac tests for the time being.  We're hitting the limit of concurrent test jobs, and I would really prefer we sort out the orbital tests before adding more requirements on the core package.

Reply:
That's fine.  Having the matrix present will help us extend more easily in the future.